evm/src/WormholeEndpoint.sol

```diff
@@ -15,7 +15,7 @@ abstract contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeRece
 
     // TODO -- fix this after some testing
     uint256 public constant GAS_LIMIT = 500000;
-    uint8 public constant CONSISTENCY_LEVEL = 1;
+    uint8 public immutable consistencyLevel;
```

Comment:
Can we add a docstring above this then to specify common consistency levels that integrators might want to use and why?